### ISSUE TITLE
Where is the shredder?

### ISSUE THREAD

**Issue by elnath78** [bleachbit/bleachbit#1501]
Pretty sure I read "install integrated shredder" while installing, but I could not find the integrated shredder in my contextual menu in file explorer. Looked in the settings there is no way to "enable" it. In addition I remember years ago I had a "safe bin" shortcut to my desktop able to shred files (this might be from another app, I'm not sure) can anyone please clarify?

**Comment by sanjaykaria** [comment]
@elnath78 
Can you please confirm whether you have installed with Admin rights, because the shredder integration with Windows Explorer needs admin rights. 
Reference https://github.com/bleachbit/bleachbit/issues/1166

**Comment by elnath78** [comment]
@sanjaykaria I installed for current user but I was on Admin user already, before BB I installed Sandboxie and it added the contextual menu entry correctly. What is the reg key fix that I can run to add BB shell integration?

**Comment by sanjaykaria** [comment]
@elnath78 
I think it is advisable to run the installer as Admin, rather than adding the registry manually (it will work) but I am not sure about the details, will need to check if it is really necessary to do it this way.

Besides, if you manually add the registry keys, the entry will remain, even after uninstalling the app, which can cause confusion.

**Comment by elnath78** [comment]
@sanjaykaria 
This makes sense, then why not ask for admin rights during the installation like other software do?

**Comment by elnath78** [comment]
@sanjaykaria 
Yes it does, but why not ask for admin rights also for the current user? Isn't this possible?

**Comment by sanjaykaria** [comment]
@elnath78 
Technically, of course, it can be done, but there needs to be a balance between installing an application with and without admin rights. 
Some users may not have admin rights and they will not be able to install the application, if the current user too needs admin rights for installing the app.
  
Besides if a standard current/user has admin rights then there is no point in differentiating between a Standard user and an Administrator.
  
There could be other reasons, maybe am not aware of.

**Comment by elnath78** [comment]
@sanjaykaria 
I was running it as sysadmin but didnt get the prompt. What I say is the installer could ask if you want the file explorer integration or not and in this case escalate, I think with UAC is possible, but not totally sure.

**Comment by rados** [comment]
@elnath78, thank you for the report and the suggestion. Generally we balance between adding features, fixing bugs, etc. If there are more requests of similar type we put more priority.
